tests/devices/qutrit_mixed/test_qutrit_mixed_measure.py

```diff
@@ -471,4 +427,176 @@ def test_variance_measurement(self, observable, ml_framework, two_qutrit_batched
         assert np.allclose(res, expected)
 
 
-# TODO TestSumOfTermsDifferentiability in future PR (with other differentiabilty tests)
+@pytest.mark.parametrize("convert_to_hamiltonian", (True, False))
```

Comment:
You can use this fixture instead:
```suggestion
@pytest.mark.usefixture("use_legacy_and_new_opmath")
```

This will automatically parametrize the test over using old and new operator arithmetic. Then you can just use `qml.Hamiltonian`, and any tensor products inside it must use `@` instead of `qml.operation.Tensor` or `qml.prod`. See [here](https://github.com/PennyLaneAI/pennylane/blob/1089bb3448081f4fde7bc3b4b792f1abda3555ff/tests/ops/functions/test_generator.py#L391) for an example.